src/components/Foilage/foilage/TreeBirch.tsx

```diff
@@ -1,13 +1,5 @@
-/*
-Auto-generated by: https://github.com/pmndrs/gltfjsx
-author: WaxFreeOintment (https://sketchfab.com/WaxFreeOintment)
-license: CC-BY-4.0 (http://creativecommons.org/licenses/by/4.0/)
-source: https://sketchfab.com/3d-models/birch-tree-voxel-459f53f6981a4482834f7700a1f2c7a8
-title: Birch Tree Voxel
-*/
-
-import {Group} from "three";
-import { useRef } from "react";
+import { Group, Color, MeshStandardMaterial } from "three";
```

Comment:
We need this comment for licensing reasons. Also, please remove the unused imports (the terminal should show errors like this)